### PR TITLE
[SERVICE-206] - Single Tab Windows w/o Tab Strip

### PR DESCRIPTION
**Pull request by JohnathonLamOF**

### CODE REVIEW

**Comment by tomrobertsOF** [comment]
Merge conflicts to fix

File: src/provider/tabbing/TabService.ts [hunk 1]
```diff
@@ -139,8 +139,22 @@ export class TabService {
      * @param {number} y Y Coordinate
      * @returns {TabGroup | null}
      */
-    public async isPointOverTabGroup(x: number, y: number): Promise<TabGroup|null> {
-        const groupTabBounds = await Promise.all(this._tabGroups.map(async group => {
+    public async isPointOverTabGroup(x: number, y: number, identity?: Identity): Promise<TabGroup|null> {
```
Comment (tomrobertsOF) [inline src/provider/tabbing/TabService.ts]: JSDoc need's updating to include the new param. Also, can the param be named something a bit more descriptive (something making it clear this identity will be ignored).

File: src/provider/tabbing/TabUtilities.ts [hunk 2]
```diff
@@ -136,7 +166,7 @@ export async function initializeTabbing(message: TabWindowOptions, uuid: string,
     }
 
     // shows the tab group window because it is default hidden
-    group.window.finWindow.show();
+    // group.window.finWindow.show();
```
Comment (tomrobertsOF) [inline src/provider/tabbing/TabUtilities.ts]: Can this commented-out code be removed

File: src/provider/tabbing/TabUtilities.ts [hunk 3]
```diff
@@ -101,9 +110,18 @@ export async function ejectTab(tabService: TabService, message: TabIdentifier&Ta
...
+                ejectedTab.ID.name,
+                tabService,
+                true);
         }
     }
+
+    if (tabGroup && tabGroup.tabs.length === 1) {
+        tabGroup.window.finWindow.hide();
+        tabGroup.tabs[0].window.updateWindowOptions({frame: true});
+    }
```
Comment (pjbroadbent) [inline src/provider/tabbing/TabUtilities.ts]: Again, duplication from having this outside of `removeTab`

File: src/provider/tabbing/TabUtilities.ts [hunk 4]
```diff
@@ -113,20 +131,32 @@ export async function ejectTab(tabService: TabService, message: TabIdentifier&Ta
...
     const tab: Tab|undefined = await group.addTab({tabID: {uuid, name}}, false, false);
 
     if (!tab) {
         console.error('No tab was added');
         return;
     }
 
+    if (ejected) {
+        tab.window.updateWindowOptions({frame: true});
+    }
```
Comment (pjbroadbent) [inline src/provider/tabbing/TabUtilities.ts]: Make this a side-effect of removing the tab?..

File: src/client/types.ts [hunk 5]
```diff
@@ -27,6 +27,7 @@ export interface WindowState extends Bounds {
 
 export interface LayoutApp {
     manifestUrl?: string;
+    manifest?: any;
```
[the remark this answers is not in the document]
Comment (MichaelMCoates) [reply src/client/types.ts]: Yeah we're no longer saving the manifest

**Comment by ryanpetersonOF** [comment]
This has been rePR'd

**Pull request closed without merging**